crates/evm/coverage/src/analysis.rs

```diff
@@ -322,33 +325,6 @@ impl<'a> ContractVisitor<'a> {
 
                 Ok(())
             }
-            NodeType::FunctionCall => {
-                self.push_item(CoverageItem {
-                    kind: CoverageItemKind::Statement,
-                    loc: self.source_location_for(&node.src),
-                    hits: 0,
-                });
-
-                let expr: Option<Node> = node.attribute("expression");
-                if let Some(NodeType::Identifier) = expr.as_ref().map(|expr| &expr.node_type) {
-                    // Might be a require/assert call
-                    let name: Option<String> = expr.and_then(|expr| expr.attribute("name"));
-                    if let Some("assert" | "require") = name.as_deref() {
-                        self.push_branches(&node.src, self.branch_id);
-                        self.branch_id += 1;
-                    }
-                }
```

Comment:
ah okay, this is now no longer required because all this did was check for assert | require